client/src/App.js

```diff
@@ -13,16 +13,28 @@ export default class App extends Component {
   }
 
   addToSavedList = movie => {
+    let newItem = true;
     const savedList = this.state.savedList;
+    savedList.map(item => {
+      if (movie.title === item.title) {
```

Comment:
In any case if you want to check to make sure the movie is not already in the list, then you can just do something like `this.state.savedList.includes(movie)` then you don't have to do the manual `for loop`